src/components/Table/TableHeaderComponent.jsx

```diff
@@ -55,6 +74,16 @@ class TableHeader extends Component {
         </th>
       );
     });
+
+    mappedVisibleColumns.unshift(
```

Comment:
It's counter intuitive that we're adding the first column at the last line of the function.